src/components/Modal/Modal.stories.tsx

```diff
@@ -111,6 +112,24 @@ export function ModalForm() {
   );
 }
 
+export function InModalState() {
```

Comment:
**nit:** I don't think this story should be necessary. We should be able to verify it works properly via tests instead. Could we remove this story to save on some snapshots?